website/content/docs/upgrading/upgrade-to-1.13.x.mdx

```diff
@@ -90,6 +90,15 @@ forwarded to the active node of this cluster.
 
 As a workaround, submit revocation requests to the active node only.
 
+### STS credentials do not return a lease_duration
+Vault 1.13.0 introduced a change to the AWS Secrets Engine such that it no longer returns leases for STS credentials due
+to the fact that they cannot be revoked or renewed. As part of this change, a bug was introduced which no longer returns
+a non-zero `lease_duration`. This prevents the Vault Agent from refreshing STS credentials and may introduce undesired behaviour
+for anything which relies on a non-zero `lease_duration`.
+
+For applications that can control what value to look for, the `ttl` value in the response can be used to know when to
+request STS credentials next.
```

Comment:
We might also want to include a mention of the `static-secret-render-interval` to control frequency for those using Vault Agent to render these secrets.